TACC/Core-CMS-Custom
# Bug/fp 2001 missing business name admins

## Overview

If user has no submitter id, business name select input is blank. If there is no submitter_id, user can't use form and should now be directed to the dashboard to create a ticket.

## Related

- [FP-2001](https://jira.tacc.utexas.edu/browse/FP-2001)


## Changes

Added new context in the view to check if sub_id is None or not. If None, render the part of the exception/extension template that asks users to submit a ticket and directs them back to the dashboard.

## Testing

1. Put in snippet on:
- line 24 of extension/views.py 
- line 33 and line 100 of exceptions/views.py
 <summary>Snippet<details>submitters = [ (2, 'TESTGOLD', 10000001, 'gmunoz1', 'CHCD'), (3, 'TESTGOLD', 10000002, 'gmunoz1', 'CHCD'), (1, 'TESTGOLD', 10000000, 'gmunoz1', 'CHCD') ]</details></summary>

2. Go to [http://localhost:8000/submissions/extension-request/](http://localhost:8000/submissions/extension-request/) and make sure the form renders and works as it did before.
3. Now, change the snippet on:
-  line 24 of extension/views.py  
 - line 33 and line 100 of exceptions/views.py
 - <summary>Snippet<details>submitters = [ ]</details></summary>
5. Make sure the screenshot below renders  and directs to "/workbench/dashboard" for both the link and the button.

## UI

Threshold Exception:
![Screen Shot 2023-03-06 at 12 57 26 PM](https://user-images.githubusercontent.com/96220951/223204799-54afdeaf-6598-459a-99f0-b42561e53c80.png)

Other Exception:
![Screen Shot 2023-03-06 at 12 58 21 PM](https://user-images.githubusercontent.com/96220951/223204990-def82bdd-6ae8-40a5-8423-95b149f86b7f.png)

Extension Request:
![Screen Shot 2023-03-06 at 12 59 30 PM](https://user-images.githubusercontent.com/96220951/223205233-db906c01-b6a8-43e8-8c26-0c5ed4f5043e.png)
